internals/models.py

```diff
@@ -1017,24 +993,15 @@ def get_in_milestone(
     all_features.extend(desktop_dev_trial_features)
     all_features.extend(android_dev_trial_features)
 
-    # Feature list must be first sorted by implementation status and then by name.
-    # The implementation may seem to be counter-intuitive using sort() method.
-    all_features.sort(key=lambda f: f.name)
-    all_features.sort(key=lambda f: f.impl_status_chrome)
-
     all_features = [f for f in all_features if not f.deleted]
-
     feature_list = [f.format_for_template() for f in all_features]
 
-    self._annotate_first_of_impl_status_in_milestones(feature_list)
-
```

Comment:
@jrobbins, these operations are not required currently. Hence, I have removed them. I will add them back if required in the next PR implementing the change you suggested.